src/space.js

```diff
@@ -65,18 +67,25 @@ class Space {
    * @param     {String}    opts.firstModerator     DID of first moderator of a thread, by default, user is first moderator
    * @param     {Boolean}   opts.members            join a members only thread, which only members can post in, defaults to open thread
    * @param     {Boolean}   opts.noAutoSub          Disable auto subscription to the thread when posting to it (default false)
+   * @param     {Boolean}   opts.ghost              Enable ephemeral messaging via Ghost Thread (optional)
    *
-   * @return    {Thread}                            An instance of the thread class for the joined thread
+   * @return    {Thread/GhostChat}                  An instance of the thread class for the joined thread
    */
   async joinThread (name, opts = {}) {
-    const subscribeFn = opts.noAutoSub ? () => {} : this.subscribeThread.bind(this)
-    if (!opts.firstModerator) opts.firstModerator = this._3id.getSubDID(this._name)
-    const thread = new Thread(namesTothreadName(this._name, name), this._replicator, this._3id, opts.members, opts.firstModerator, subscribeFn)
-    const address = await thread._getThreadAddress()
-    if (this._activeThreads[address]) return this._activeThreads[address]
-    await thread._load()
-    this._activeThreads[address] = thread
-    return thread
+    if (opts.ghost) {
+      const chat = new GhostChat(namesToChatName(name), this._replicator, this._3id)
```

Comment:
I would suggest the following change as well:
```suggestion
      const ghostAddress = namesToChatName(this._name, name)
      if (!this._activeThreads[ghostAddress]) {
          this._activeThreads[ghostAddress] = new GhostChat(ghostAddress, this._replicator, this._3id)
      }
      return this._activeThreads[ghostAddress]
```